### PR TITLE
Added global variables. Created the basic desktop structure. Began st…

### PR DESCRIPTION
**Pull request by bobbypwang**
…yling the Navigation.

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -5,10 +5,13 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <meta name="theme-color" content="#000000" />
+    
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,400i,500,700&display=swap" rel="stylesheet">
+
     <script src="https://apis.google.com/js/api.js"></script>
     <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
-    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css" />
+    <!-- <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css" /> -->
```
Comment (AndresMWeber) [inline client/public/index.html:14]: comment

File: client/src/components/pages/Home.jsx [hunk 2]
```diff
@@ -37,6 +37,7 @@ class Home extends React.Component {
 
   render() {
     return (
+      <React.Fragment>
```
Comment (AndresMWeber) [inline client/src/components/pages/Home.jsx]: Should refactor this since it doesn't need a Fragment

**Review by AndresMWeber — APPROVED** [review]

File: client/src/styles/_global.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+body {
+  font-family: $fontFamily;
+}
+
+a {
+    color: $purple;
```
Comment (Kiwi-x-Kiwi) [inline client/src/styles/_global.scss:6]: Extra spacing...  Reeeee!!!

File: client/src/styles/_variables.scss [hunk 4]
```diff
@@ -1,2 +1,22 @@
...
+// Used to calculate the column classes in _grid.scss
 $columnNumber: 12;
 $columnWidth: (1 / $columnNumber);
+
+// Helper spacing classes
+$tiny: 3px;
+$small: 7px;
+$medium: 15px;
+$large: 27px;
+$xlarge: 42px;
```
Comment (Kiwi-x-Kiwi) [inline client/src/styles/_variables.scss:22]: Should we use xlarge or xLarge?

**Review by Kiwi-x-Kiwi — APPROVED** [review]

**Pull request merged**